templates/repo/diff/options_dropdown.tmpl

```diff
@@ -0,0 +1,9 @@
+<div class="ui dropdown tiny button">
+	{{.i18n.Tr "repo.diff.options_button"}}
+	<i class="dropdown icon"></i>
+	<div class="menu">
+		<a class="item tiny basic toggle button" data-target="#diff-files">{{.i18n.Tr "repo.diff.show_diff_stats"}}</a>
+		<a class="item" href="{{$.RepoLink}}/pulls/{{.Issue.Index}}.patch" download="15.patch">{{.i18n.Tr "repo.diff.download_patch"}}</a>
```

Comment:
What does 15 stand for here? (and the line below)

Reply:
O you got me!

Reply:
-> 281715d082a8493ca29e50b93dab46693375de27

Reply:
@guillep2k was testing the html5 property "download"